services/ui-src/src/components/sections/Userinfo.jsx

```diff
@@ -1,8 +1,11 @@
 import React from "react";
-import PropTypes from "prop-types";
-import { connect } from "react-redux";
+import { shallowEqual, useSelector } from "react-redux";
 
-const Userinfo = ({ currentUser }) => {
+const Userinfo = () => {
+  const currentUser = useSelector(
+    (state) => state.stateUser.currentUser,
+    shallowEqual
+  );
```

Comment:
```suggestion
  const currentUser = useSelector((state) => state.stateUser.currentUser);
```

This can just be a straight call, I've been using the shallowEqual only so I don't have to write out multiple useSelector calls and can combine them into one thing.

Reply:
Good to know! I should have looked at what it was doing instead of just copy-pasting 😅